openhands_ext/test_storage.py

```
@@ -0,0 +1,203 @@
+    
+    def __post_init__(self):
+        if self.api_keys is None:
+            self.api_keys = {}
+
+class TestStorageManager:
+    """
+    Test storage manager that demonstrates multi-tenant storage patterns
+    In enterprise, this would be a database with proper encryption
+    """
+    
+    def __init__(self, base_path: str = "/tmp/openhands-test-storage"):
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/openhands-test-storage"

(S108)